dataservice/api/study/models.py

```diff
@@ -29,6 +30,7 @@ class Study(db.Model, Base):
     version = db.Column(db.Text())
     name = db.Column(db.Text())
     attribution = db.Column(db.Text())
+    release_status = db.Column(db.Text())
```

Comment:
Maybe eventually want to make this an enum

Reply:
yeah, updated in enum issue #48